rust error when docker start

I am deploying the application using docker compose without changing any configuration, and  a Rust error during the startup process. Since I am not familiar with Rust, I cannot figure out what happened. The full log is as follows.
```
[2024-Sep-10 10:13:54.335833] Starting deployment of metz-sh to experimental.
[2024-Sep-10 10:13:54.493878] Preparing container with helper image: ghcr.io/coollabsio/coolify-helper:latest.
[2024-Sep-10 10:13:54.582961]

[COMMAND] docker rm -f l4ook44o48wskwsocwoosws8
[OUTPUT]
Error response from daemon: No such container: l4ook44o48wskwsocwoosws8

[2024-Sep-10 10:13:54.694669]

[COMMAND] docker run -d --network a8wo48s --name l4ook44o48wskwsocwoosws8 --rm -v /var/run/docker.sock:/var/run/docker.sock ghcr.io/coollabsio/coolify-helper:latest
[OUTPUT]
401f468893013b62b04b5444eaacd575d022753997176d9e4828ea5c766fb5c1

[2024-Sep-10 10:13:56.242705]

[COMMAND] docker exec l4ook44o48wskwsocwoosws8 bash -c 'GIT_SSH_COMMAND="ssh -o ConnectTimeout=30 -p 22 -o Port=22 -o LogLevel=ERROR -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null" git ls-remote https://github.com/metz-sh/simulacrum-self-host main'
[OUTPUT]
67c29008f438209985259600fdc98af251d30905 refs/heads/main

[2024-Sep-10 10:13:56.250516] ----------------------------------------
[2024-Sep-10 10:13:56.251463] Importing metz-sh/simulacrum-self-host:main (commit sha HEAD) to /artifacts/l4ook44o48wskwsocwoosws8.
[2024-Sep-10 10:13:56.377406]

[COMMAND] docker exec l4ook44o48wskwsocwoosws8 bash -c 'git clone -b "main" https://github.com/metz-sh/simulacrum-self-host /artifacts/l4ook44o48wskwsocwoosws8 && cd /artifacts/l4ook44o48wskwsocwoosws8 && sed -i "s#git@\(.*\):#https://\1/#g" /artifacts/l4ook44o48wskwsocwoosws8/.gitmodules || true && cd /artifacts/l4ook44o48wskwsocwoosws8 && GIT_SSH_COMMAND="ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null" git submodule update --init --recursive && cd /artifacts/l4ook44o48wskwsocwoosws8 && GIT_SSH_COMMAND="ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null" git lfs pull'
[OUTPUT]
Cloning into '/artifacts/l4ook44o48wskwsocwoosws8'...

[2024-Sep-10 10:14:02.058454]

[COMMAND] docker exec l4ook44o48wskwsocwoosws8 bash -c 'git clone -b "main" https://github.com/metz-sh/simulacrum-self-host /artifacts/l4ook44o48wskwsocwoosws8 && cd /artifacts/l4ook44o48wskwsocwoosws8 && sed -i "s#git@\(.*\):#https://\1/#g" /artifacts/l4ook44o48wskwsocwoosws8/.gitmodules || true && cd /artifacts/l4ook44o48wskwsocwoosws8 && GIT_SSH_COMMAND="ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null" git submodule update --init --recursive && cd /artifacts/l4ook44o48wskwsocwoosws8 && GIT_SSH_COMMAND="ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null" git lfs pull'
[OUTPUT]
sed: /artifacts/l4ook44o48wskwsocwoosws8/.gitmodules: No such file or directory

[2024-Sep-10 10:14:02.502985]

[COMMAND] docker exec l4ook44o48wskwsocwoosws8 bash -c 'cd /artifacts/l4ook44o48wskwsocwoosws8 && git log -1 67c29008f438209985259600fdc98af251d30905 --pretty=%B'
[OUTPUT]
Update README.md

[2024-Sep-10 10:14:08.137231] Pulling & building required images.
[2024-Sep-10 10:14:08.644789]

[COMMAND] docker exec l4ook44o48wskwsocwoosws8 bash -c 'SOURCE_COMMIT=67c29008f438209985259600fdc98af251d30905 COOLIFY_BRANCH=main docker compose --env-file /artifacts/l4ook44o48wskwsocwoosws8/.env --project-name g880sc48osg0ws8ggwgw808w --project-directory /artifacts/l4ook44o48wskwsocwoosws8 -f /artifacts/l4ook44o48wskwsocwoosws8/docker-compose.yaml build --pull'
[OUTPUT]
#0 building with "default" instance using docker driver

#1 [server internal] load build definition from Dockerfile
#1 transferring dockerfile: 1.00kB done
#1 DONE 0.0s

#2 [server internal] load metadata for docker.io/lukemathwalker/cargo-chef:0.1.66-rust-slim-bookworm

[2024-Sep-10 10:14:10.465199]

[COMMAND] docker exec l4ook44o48wskwsocwoosws8 bash -c 'SOURCE_COMMIT=67c29008f438209985259600fdc98af251d30905 COOLIFY_BRANCH=main docker compose --env-file /artifacts/l4ook44o48wskwsocwoosws8/.env --project-name g880sc48osg0ws8ggwgw808w --project-directory /artifacts/l4ook44o48wskwsocwoosws8 -f /artifacts/l4ook44o48wskwsocwoosws8/docker-compose.yaml build --pull'
[OUTPUT]
#2 ...

#3 [server internal] load metadata for docker.io/library/debian:trixie-slim
#3 DONE 2.0s

[2024-Sep-10 10:14:10.640669]

[COMMAND] docker exec l4ook44o48wskwsocwoosws8 bash -c 'SOURCE_COMMIT=67c29008f438209985259600fdc98af251d30905 COOLIFY_BRANCH=main docker compose --env-file /artifacts/l4ook44o48wskwsocwoosws8/.env --project-name g880sc48osg0ws8ggwgw808w --project-directory /artifacts/l4ook44o48wskwsocwoosws8 -f /artifacts/l4ook44o48wskwsocwoosws8/docker-compose.yaml build --pull'
[OUTPUT]
#2 [server internal] load metadata for docker.io/lukemathwalker/cargo-chef:0.1.66-rust-slim-bookworm
#2 DONE 2.0s

#4 [server internal] load .dockerignore
#4 transferring context: 112B done
#4 DONE 0.0s

#5 [server runtime 1/5] FROM docker.io/library/debian:trixie-slim@sha256:43788d0fe1fee841d09707fbb9355c938c6eb5b0c6bde2de31a07443f1bc6750
#5 DONE 0.0s

#6 [server runtime 2/5] WORKDIR app
#6 CACHED

#7 [server runtime 3/5] RUN apt-get update
#7 CACHED

#8 [server runtime 4/5] RUN apt-get -y install openssl libssl-dev ca-certificates curl
#8 CACHED

#9 [server chef 1/2] FROM docker.io/lukemathwalker/cargo-chef:0.1.66-rust-slim-bookworm@sha256:2ca0f1f2e81dff0ab84bddca351d55561916e8cc1c23bfa7d99e7eebd8daa3e1
#9 DONE 0.0s

#10 [server internal] load build context
#10 transferring context: 216.77kB done
#10 DONE 0.0s

#11 [server builder 1/6] COPY --from=planner /app/recipe.json recipe.json
#11 CACHED

#12 [server planner 2/2] RUN cargo chef prepare --recipe-path recipe.json
#12 CACHED

#13 [server builder 3/6] RUN apt-get -y install libssl-dev pkg-config libpkgconf3 pkgconf pkgconf-bin
#13 CACHED

#14 [server builder 4/6] RUN cargo chef cook --release --recipe-path recipe.json
#14 CACHED

#15 [server chef 2/2] WORKDIR app
#15 CACHED

#16 [server builder 2/6] RUN apt-get update
#16 CACHED

#17 [server planner 1/2] COPY . .
#17 CACHED

#18 [server builder 5/6] COPY . .
#18 CACHED

#19 [server builder 6/6] RUN cargo build --release --bin rusty-backend

[2024-Sep-10 10:14:10.907132]

[COMMAND] docker exec l4ook44o48wskwsocwoosws8 bash -c 'SOURCE_COMMIT=67c29008f438209985259600fdc98af251d30905 COOLIFY_BRANCH=main docker compose --env-file /artifacts/l4ook44o48wskwsocwoosws8/.env --project-name g880sc48osg0ws8ggwgw808w --project-directory /artifacts/l4ook44o48wskwsocwoosws8 -f /artifacts/l4ook44o48wskwsocwoosws8/docker-compose.yaml build --pull'
[OUTPUT]
#19 0.418 Compiling rusty-backend v0.1.0 (/app)

[2024-Sep-10 10:14:11.223487]

[COMMAND] docker exec l4ook44o48wskwsocwoosws8 bash -c 'SOURCE_COMMIT=67c29008f438209985259600fdc98af251d30905 COOLIFY_BRANCH=main docker compose --env-file /artifacts/l4ook44o48wskwsocwoosws8/.env --project-name g880sc48osg0ws8ggwgw808w --project-directory /artifacts/l4ook44o48wskwsocwoosws8 -f /artifacts/l4ook44o48wskwsocwoosws8/docker-compose.yaml build --pull'
[OUTPUT]
#19 0.734 error[E0599]: no method named `build_sqlx` found for mutable reference `&mut sea_query::UpdateStatement` in the current scope
#19 0.734 --> src/repositories/project_repository.rs:155:10
#19 0.734 |
#19 0.734 144 | let (sql, values) = Query::update()
#19 0.734 | _____________________________-
#19 0.734 145 | | .table(TableRef::Table(SIden(TABLE).into_iden()))
#19 0.734 146 | | .values(update_values)
#19 0.734 147 | | .and_where(Expr::col(ProjectIden::Id).eq(project_id))
#19 0.734 ... |
#19 0.734 154 | | ]))
#19 0.734 155 | | .build_sqlx(PostgresQueryBuilder);
#19 0.734 | |_________-^^^^^^^^^^
#19 0.734 |
#19 0.734 help: there is a method `build` with a similar name
#19 0.734 |
#19 0.734 155 | .build(PostgresQueryBuilder);
#19 0.734 | ~~~~~
#19 0.734

[2024-Sep-10 10:14:11.375927]

[COMMAND] docker exec l4ook44o48wskwsocwoosws8 bash -c 'SOURCE_COMMIT=67c29008f438209985259600fdc98af251d30905 COOLIFY_BRANCH=main docker compose --env-file /artifacts/l4ook44o48wskwsocwoosws8/.env --project-name g880sc48osg0ws8ggwgw808w --project-directory /artifacts/l4ook44o48wskwsocwoosws8 -f /artifacts/l4ook44o48wskwsocwoosws8/docker-compose.yaml build --pull'
[OUTPUT]
#19 0.736 error[E0277]: the size for values of type `str` cannot be known at compilation time
#19 0.736 --> src/repositories/project_repository.rs:144:14
#19 0.736 |
#19 0.736 144 | let (sql, values) = Query::update()
#19 0.736 | ^^^ doesn't have a size known at compile-time
#19 0.736 |
#19 0.736 = help: the trait `Sized` is not implemented for `str`
#19 0.736 = note: all local variables must have a statically known size
#19 0.736 = help: unsized locals are gated as an unstable feature
#19 0.736

[2024-Sep-10 10:14:11.418787]

[COMMAND] docker exec l4ook44o48wskwsocwoosws8 bash -c 'SOURCE_COMMIT=67c29008f438209985259600fdc98af251d30905 COOLIFY_BRANCH=main docker compose --env-file /artifacts/l4ook44o48wskwsocwoosws8/.env --project-name g880sc48osg0ws8ggwgw808w --project-directory /artifacts/l4ook44o48wskwsocwoosws8 -f /artifacts/l4ook44o48wskwsocwoosws8/docker-compose.yaml build --pull'
[OUTPUT]
#19 0.930 warning: unused import: `sea_query_binder::SqlxBinder`
#19 0.930 --> src/repositories/project_repository.rs:4:5
#19 0.930 |
#19 0.930 4 | use sea_query_binder::SqlxBinder;
#19 0.930 | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
#19 0.930 |
#19 0.930 = note: `#[warn(unused_imports)]` on by default
#19 0.930

[2024-Sep-10 10:14:11.572356]

[COMMAND] docker exec l4ook44o48wskwsocwoosws8 bash -c 'SOURCE_COMMIT=67c29008f438209985259600fdc98af251d30905 COOLIFY_BRANCH=main docker compose --env-file /artifacts/l4ook44o48wskwsocwoosws8/.env --project-name g880sc48osg0ws8ggwgw808w --project-directory /artifacts/l4ook44o48wskwsocwoosws8 -f /artifacts/l4ook44o48wskwsocwoosws8/docker-compose.yaml build --pull'
[OUTPUT]
#19 1.083 Some errors have detailed explanations: E0277, E0599.
#19 1.083 For more information about an error, try `rustc --explain E0277`.

[2024-Sep-10 10:14:11.734911]

[COMMAND] docker exec l4ook44o48wskwsocwoosws8 bash -c 'SOURCE_COMMIT=67c29008f438209985259600fdc98af251d30905 COOLIFY_BRANCH=main docker compose --env-file /artifacts/l4ook44o48wskwsocwoosws8/.env --project-name g880sc48osg0ws8ggwgw808w --project-directory /artifacts/l4ook44o48wskwsocwoosws8 -f /artifacts/l4ook44o48wskwsocwoosws8/docker-compose.yaml build --pull'
[OUTPUT]
#19 1.095 warning: `rusty-backend` (lib) generated 1 warning
#19 1.095 error: could not compile `rusty-backend` (lib) due to 2 previous errors; 1 warning emitted

[2024-Sep-10 10:14:11.797384]

[COMMAND] docker exec l4ook44o48wskwsocwoosws8 bash -c 'SOURCE_COMMIT=67c29008f438209985259600fdc98af251d30905 COOLIFY_BRANCH=main docker compose --env-file /artifacts/l4ook44o48wskwsocwoosws8/.env --project-name g880sc48osg0ws8ggwgw808w --project-directory /artifacts/l4ook44o48wskwsocwoosws8 -f /artifacts/l4ook44o48wskwsocwoosws8/docker-compose.yaml build --pull'
[OUTPUT]
#19 ERROR: process "/bin/sh -c cargo build --release --bin rusty-backend" did not complete successfully: exit code: 101

[2024-Sep-10 10:14:11.811804]

[COMMAND] docker exec l4ook44o48wskwsocwoosws8 bash -c 'SOURCE_COMMIT=67c29008f438209985259600fdc98af251d30905 COOLIFY_BRANCH=main docker compose --env-file /artifacts/l4ook44o48wskwsocwoosws8/.env --project-name g880sc48osg0ws8ggwgw808w --project-directory /artifacts/l4ook44o48wskwsocwoosws8 -f /artifacts/l4ook44o48wskwsocwoosws8/docker-compose.yaml build --pull'
[OUTPUT]
------
> [server builder 6/6] RUN cargo build --release --bin rusty-backend:
0.930 |
0.930 4 | use sea_query_binder::SqlxBinder;
0.930 | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
0.930 |
0.930 = note: `#[warn(unused_imports)]` on by default
0.930
1.083 Some errors have detailed explanations: E0277, E0599.
1.083 For more information about an error, try `rustc --explain E0277`.
1.095 warning: `rusty-backend` (lib) generated 1 warning
1.095 error: could not compile `rusty-backend` (lib) due to 2 previous errors; 1 warning emitted
------

[2024-Sep-10 10:14:11.813442]

[COMMAND] docker exec l4ook44o48wskwsocwoosws8 bash -c 'SOURCE_COMMIT=67c29008f438209985259600fdc98af251d30905 COOLIFY_BRANCH=main docker compose --env-file /artifacts/l4ook44o48wskwsocwoosws8/.env --project-name g880sc48osg0ws8ggwgw808w --project-directory /artifacts/l4ook44o48wskwsocwoosws8 -f /artifacts/l4ook44o48wskwsocwoosws8/docker-compose.yaml build --pull'
[OUTPUT]
failed to solve: process "/bin/sh -c cargo build --release --bin rusty-backend" did not complete successfully: exit code: 101

[2024-Sep-10 10:14:11.847484] Oops something is not okay, are you okay? 😢

[2024-Sep-10 10:14:11.849250] failed to solve: process "/bin/sh -c cargo build --release --bin rusty-backend" did not complete successfully: exit code: 101
```

--- a/rusty-backend/Cargo.toml
+++ b/rusty-backend/Cargo.toml
@@ -29,7 +29,7 @@ tracing-subscriber = { version = "0.3", features = ["env-filter", "json"] }
 url = {version = "2.5.0", features = ["serde"]}
 sea-query={ version = "0.30.7", features = ["derive", "attr", "backend-postgres", "with-json", "with-uuid"]}
 modql={ version = "0.3.10", features = ["with-sea-query"]}
-sea-query-binder = { version = "0", features = [
+sea-query-binder = { version = "0.5.0", features = [
     "sqlx-postgres",
     "with-json",
     "with-uuid"

